website/server/models/task.js

```diff
@@ -131,6 +131,10 @@ TaskSchema.plugin(baseModel, {
       delete taskObj.value;
     }
 
+    if (taskObj.priority && !Number.isNaN(Number.parseFloat(taskObj.priority))) {
+      taskObj.priority = Number(taskObj.priority.toFixed(1));
```

Comment:
shouldn't this line be `Number(taskObj.priority).toFixed(1);` since toFixed works on numbers, so it makes sense that you first make sure it's a number with Number and then call the toFixed method on the number